src/ServiceInsight/ValueConverters/SearchStringConverter.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace ServiceInsight.ValueConverters
+{
+    using System;
+    using System.Globalization;
+    using System.Windows.Data;
+
+    public class SearchStringConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var str = string.Format("{0}", value);
+            if (str.StartsWith("\"") && str.EndsWith("\""))
+            {
+                str = str.Substring(1, str.Length - 2);
+            }
+
+            return str;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return string.Format("\"{0}\"", value);
+        }
```

Comment:
You're both right! You could use exceptions in a converter and they show up as warnings, but the right approach is to use the `DependencyProperty.UnsetValue` as @WilliamBZA highlighted.